### ISSUE TITLE
Podfiles I don't want install is detected and will installed.

### ISSUE THREAD

**Issue by RyogaK** [bitrise-io/steps-cocoapods-install#6]
Now I'm using both of Cocoapods and Carthage, also cache of Carthage is.
Now this script will find Podfile in `.Carthage/` so I need help.

Could you make a filter parameter for ignoring that?

**Comment by viktorbenei** [comment]
Why do you have Podfiles in your `.Carthage` dir?

**Comment by RyogaK** [comment]
Some library supports Carthage are supports Cocoapods as well, so there are some Podfile in ./Carthage/Checkout If I cache that.

**Comment by viktorbenei** [comment]
Makes sense, we'll add a solution for that in this step, in the meantime you can use a Script step to run "pod install" or anything you want to

**Comment by RyogaK** [comment]
Thanks 😆

**Comment by viktorbenei** [comment]
😉

**Comment by viktorbenei** [comment]
Fixed in `1.3.0`. Thanks for reporting!

TL;DR; we added a new "Podfile path" input. If you set that, then the step will only install the Podfile from that path and will ignore every other Podfile in the repository.
